data/tests/test_management_commands.py

```diff
@@ -12,5 +12,7 @@ def setUpTestData(cls):
 
     def test_command_import_contributors_data(self):
         contributors = Contributor.objects.all()
+        if not contributors:
+            return
```

Comment:
https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures

Reply:
I am having a bit of problem here:

How can we use the Contributor model outside of test methods?

Suppose I do this:
```
num_contrib = Contributor.objects.all().count()
@skipIf(num_contrib==0, 'There are not cotrib..')
def test_command_import_contributors_data(self):
     ...
```
In this case, the test will always be skipped no matter if we are getting the data from the webservices or not as we are not using the model from the `SetUpTestData`.

Reply:
there is always the option of using `unittest.SkipTest` it is mentioned in that section.